### ISSUE TITLE
Controls need labels for screen readers.

### ISSUE THREAD

**Issue by rkingett** [yermak/AudioBookConverter#530]
I am a Blind user running the [free and open source NVDA screen reader on Windows,](https://www.nvaccess.org/download/) the latest version of Windows11. I like the program, but it is quite difficult to use at the moment because a number of the controls don't appear to have labels associated with the control.

The most obvious place I can think of to test this is in the qualities tab.

NVDA reads the stile of control, combo box, slider, and it's value, but it doesn't read the labels associated with that control.

There are too many to list, in terms of problematic areas. This is a really great tool and would love to help make it more accessible to everyone.

**Comment by yermak** [comment]
Version 6.5.0 contains a lot of fixes for tooltips. I will close this issue for now, feel free to provide feedback how it works.
